ci/bump_version.sh

```diff
@@ -1,51 +1,63 @@
-#!/bin/bash -xv
+#!/bin/bash
 
 if [ $# != 1 ]
 then
      echo "Please provide version as the first argument"
      exit 1
 fi
 
-version=$1
+echo "Searching for crates..."
+
+crates=($(find . -type d -exec sh -c '[ -f "$0"/Cargo.toml ]' '{}' \; -print))
+
+if [[ "${#crates[@]}" == "0" ]]; then
+  echo "No crates found."
+  exit 1
+else
+  echo "Found ${#crates[@]} crate(s): ${crates[*]}"
+fi
+
+version="$1"
 CI_DIR="$( cd "$( dirname "$0" )" && pwd )"
```

Comment:
If you do this at the beginning, then you can use it as a reference to `cd` to where the crates are, rather than depending on the user to run this script from where the crates are.